src/server/HtmlDocument.js

```diff
@@ -227,6 +230,19 @@ class HtmlDocument extends React.Component {
             name="msapplication-TileImage"
             content="/static/favicons/mstile-144x144.png"
         />
+
+        <script dangerouslySetInnerHTML={{__html: envConfig}} />
+        { GA_TRACKING_CODE &&
```

Comment:
Not on browsers that support `async` which is everything remotely recent (hence google recommending it)